website_lazy_load_image/tests/test_lazy_load_image.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Onestein
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+from lxml import etree
+
+
+class DateRangeTest(TransactionCase):
```

Comment:
pls, use `SavepointCase` and replace `setUp` w/ `setUpClass`: will make test faster